lib/drivers/redis.js

```diff
@@ -110,6 +110,9 @@
                 data[data.length - 1] = data[data.length - 1].map(function (index) {
                     return parseInt(index, 10);
                 });
+                data.forEach(function (row, index) {
```

Comment:
Would the request just fall on the floor, or would they get routed to a weird location, probably ending in some 5xx?